site/about.php

```diff
@@ -5,30 +5,105 @@
-                <img class="img-fluid" src="assets/images/about-robot.png" alt="robot">
+            <div class="col-sm order-sm-first my-auto">
+                <h2 class="mb-3">Data without Insight isn't worth much.</h2>
+                <p>We are excited about data science and analytics for learning. Games provide rich descriptions into how learners interact with complex systems. We are developing new methods to transform that raw "click steam" data into insights that can be used by researchers, designers, and educators. Open Game Data is the infrastructure that will support those discoveries.</p>
             </div>
         </div>
-        <div class="row top-buffer">
-            <div class="col-sm my-auto order-sm-last">
-                <h2>Data without Insight isn't worth much.</h2>
-                <p>Everyone is excited about data science and analytics for learning. Games provide rich descriptions into how learners interact with complex systems. We are excited to develop new methods to transform that raw &quot;click steam&quot; data into insights that can be used by designers, educators, researchers and learners. Open Game Data is the infrastructure that will support those discoveries.</p>
-            </div>            
-            <div class="col-sm order-sm-first about-data my-auto">
-                <img class="img-fluid" src="assets/images/about-data.png" alt="data graph">
-            </div>
-
+        <h2 class="text-sm-center py-4 mx-md-5">Research Insights are only valuable if you can access them:</h2>
+        <div class="row row-cols-sm-2 blue-grad text-dark mb-5">
+            <div class="col-sm text-center p-5">
+                <img class="mb-2" src="assets/images/graphics/open-data.png" alt="">
+                <h3 class="h4">Open Data:</h3>
+                <p class="lead mx-3 mx-sm-0 mx-md-4">We share the raw data produced as part of an experiment.</p>
+            </div>
+            <div class="col-sm text-center p-5">
+                <img class="mb-2" src="assets/images/graphics/open-analysis.png" alt="">
+                <h3 class="h4">Open Analysis:</h3>
+                <p class="lead mx-3 mx-sm-0 mx-md-4">Access to code that shows how we do statistical analysis on the data associated with each publication.</p>
+            </div>
+            <div class="col-sm text-center p-5">
+                <img class="mb-2" src="assets/images/graphics/open-material.png" alt="">
+                <h3 class="h4">Open Materials:</h3>
+                <p class="lead mx-3 mx-sm-0 mx-md-4">Access to game source code, assessment and interview instruments.</p>
+            </div>
+            <div class="col-sm text-center p-5">
+                <img class="mb-2" src="assets/images/graphics/open-access.png" alt="">
+                <h3 class="h4">Open Access:</h3>
+                <p class="lead mx-3 mx-sm-0 mx-md-4">We encourage resear chers to make their research outputs available (freely) to everyone.</p>
```

Comment:
There is an extra space in "researchers". It is in the mock-up as well but I think it must be a typo. 
<img width="147" alt="image" src="https://github.com/opengamedata/opengamedata-website/assets/46722573/5068a499-cea0-4d57-bbdc-3d9a7d38000e">